fab/environments.yml

```diff
@@ -143,7 +143,7 @@ production:
       email_queue:
         concurrency: 2
       repeat_record_queue:
-        concurrency: 3
+        concurrency: 20
```

Comment:
I guess it is. i forget how it works but this doesn't actually make 20 celery processes, just 20 gevent forks?